examples/url_shortener/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "@tezos/url_shortener",
+  "authors": "TriliTech Ecosystems and Tools Team <contact@trili.tech>",
+  "private": true,
+  "version": "0.0.0",
+  "main": "index.ts",
+  "dependencies": {
+    "@tezos/jstz": "^0.0.0"
+  },
+  "devDependencies": {
+    "esbuild": "^0.20.1"
+  },
+  "scripts": {
+    "build": "esbuild index.ts --bundle --format=esm --target=esnext --minify --outfile=dist/index.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "description": ""
```

Comment:
nit: some dummy package.json entries that look like they could be removed (or filled in)

Additionally our license isn't ISC but MIT